File: Modul 3/Soal 4/PRAK304-NIM-Nama.c
#include <stdio.h>

int main(){
    int inp;

    printf("Input: ");
    scanf("%d", &inp);

    if (inp == 0){
        printf("Nol");
    }
    else if (inp >= 1 && inp <= 9){
        printf("Satuan");
    }
    else if (inp == 10){
        printf("Puluhan");
    }
    else if (inp >= 11 && inp <= 19){
        printf("Belasan");
    }
    else if (inp >= 20 && inp <= 99){
        printf("Puluhan");
    }
    else printf("Anda Menginput Melebihi Limit Bilangan");
}

File: Modul 3/Soal 4/PRAK304-NIM-Nama.py
inp = int(input("Input: "))

if inp == 0:
    print("Nol")
elif inp >= 1 and inp <= 9:
    print("Satuan")
elif inp == 10:
    print("Puluhan")
elif inp >= 11 and inp <= 19:
    print("Belasan")
elif inp >= 20 and inp <= 99:
    print("Puluhan")
else:
    print("Anda Menginput Melebihi Limit Bilangan")